sw/device/mask_rom/mask_rom.c

```diff
@@ -3,13 +3,43 @@
 // SPDX-License-Identifier: Apache-2.0
 
 #include "sw/device/mask_rom/mask_rom.h"
+
+#include <stdbool.h>
+#include <stdint.h>
+
 #include "sw/device/lib/base/stdasm.h"
+#include "sw/device/lib/runtime/hart.h"
+#include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"
+
+// This is the expected ROM_EXT Manifest Identifier
+static uint32_t const kRomExtIdentifierExpected = 0x4552544F;
+
+// These are dummy definitions while we await the ROM_EXT Manifest parser and
+// the boot policy module. This only allows us to boot a ROM_EXT image from
+// "Slot A" (the bottom half of flash).
+static void *const kRomExtSlotAAddr = (void *)TOP_EARLGREY_EFLASH_BASE_ADDR;
+
+static uint32_t *const kRomExtSlotAIdentifier =
+    (uint32_t *)(kRomExtSlotAAddr + 0x0u);
+static void *const kRomExtSlotAIrqVector = (void *)(kRomExtSlotAAddr + 0x400u);
```

Comment:
I suppose I should have a getter in ROM_EXT parser for this as well?

Reply:
That would be helpful, though this is not a field per-se.